classification1.Rmd

```diff
@@ -163,13 +157,14 @@ glimpse(cancer)
 ```
 
 Recall factors have what are called "levels", which you can think of as categories. We
-can ask for the levels from the `Class` column by using the `levels` function.
+can verify the levels of the `Class` column by using the `levels` function.
 This function should return the name of each category in that column. Given
-that we only have 2 different values in our `Class` column (B and M), we
-only expect to get two names back.  Note that the `levels` function requires
-a *vector* argument, while the `select` function outputs a *data frame*; 
-so we use the `pull` function, which converts a single
-column of a data frame into a vector.
+that we only have two different values in our `Class` column (B for benign and M 
+for malignant), we only expect to get two names back.  Note that the `levels` function requires
+a *vector* argument (while the `select` function outputs a *data frame*); 
```

Comment:
why does this part mention the `select` function? This must be based on some old code we had here...